Fixed issues regarding AutoComplete and integrated Crashlytics.

teodorciuraru opened the pull request:
**Changelog:**
- Fixed “Remind me at” UI bug.
- Added constraint to the Plan my Trip screen that will add some space between the title and the Peace Corps logo.
- Fixed Autocomplete deprecated API Key.
- Removed Autocomplete button that triggered the search. Now it triggers by pressing on the TextField.
- Installed new Cocoapods.
- Fixed Autocomplete bug on iOS versions lesser than 9.0.
- Adds Fabric's Crashlytics and Answers.

Commit:
Adds the following changes:

- Fixed “Remind me at” UI bug.
- Added constraint to the Plan my Trip screen that will add some space between the title and the Peace Corps logo.
- Fixed Autocomplete deprecated API Key.
- Removed Autocomplete button that triggered the search. Now it triggers by pressing on the TextField.
- Installed new Cocoapods.
- Fixed Autocomplete bug on iOS versions lesser than 9.0.

bphenriques commented on Malaria App Widget/TodayViewController.swift:
```diff
@@ -23,7 +23,8 @@ class TodayViewController: UIViewController, NCWidgetProviding {
   // Solves unnecesary widget bottom margin problem
-  func widgetMarginInsetsForProposedMarginInsets(var defaultMarginInsets: UIEdgeInsets) -> UIEdgeInsets {
+  func widgetMarginInsetsForProposedMarginInsets(defaultMarginInsets: UIEdgeInsets) -> UIEdgeInsets {
+    var defaultMarginInsets = defaultMarginInsets
```
Isn't this a superfluous line?

teodorciuraru replied:
That line solves attributing a value to a variable function parameter (on the next line), making the code compatible with Swift 3, where "var paramaters" are not allowed anymore.

I followed this suggestion and I am open to any other alternatives.

https://github.com/apple/swift-evolution/blob/master/proposals/0003-remove-var-parameters.md

bphenriques commented on Podfile:
```diff
@@ -3,15 +3,29 @@ xcodeproj 'malaria-ios.xcodeproj'
-pod 'SwiftyJSON', '~> 2.3.1'
-pod 'Charts', '~> 2.1.6'
-pod 'CVCalendar', '~> 1.2.8'
+def testing_pods
```
Change name, "testing-pods" is not suitable unless it really is. testing pods would make more sense to target malaria-iosTests.

Moreover, Charts, CVCalendar and the remaining UI views can be on a different `def test-dependencies` since they are not required for unit testing.

bphenriques replied:
the second comment is not critical but take a look into it.

teodorciuraru replied:
As far as I can browse the code, there is no framework required for malaria_tests. Am I right?

bphenriques replied:
There is. Alamofire and SwiftyJson are required to test the info hub

bphenriques commented on malaria-ios/ViewControllers/SetupScreen/SetupScreenViewController.swift:
```diff
@@ -82,14 +83,18 @@ extension SetupScreenViewController {
             medicineAlert.addAction(UIAlertAction(title: AlertOptions.ok, style: .Destructive, handler: { _ in
+
+                // Send Answers (Crashlytics) log to server
+                Answers.logCustomEventWithName("User changed pill.", customAttributes: ["Medicine Name" : current.name])
```
cool! 👍

teo029 commented:
That line solves attributing a value to a variable function parameter (on the next line), making the code compatible with Swift 3, where "var paramaters" are not allowed anymore.

I followed this suggestion and I am open to any other alternatives.

https://github.com/apple/swift-evolution/blob/master/proposals/0003-remove-var-parameters.md

Commit:
Change PodFile and add Reachability

Added Reachability to check internet connection on triggering AutoComplete.
Removed the Malaria Test target from the Podfile.
Added new app iPad icon.

Commit:
Removed copyright comments
